pkg/reconcile/pipeline/context/impl_test.go

```diff
@@ -91,6 +92,51 @@ var _ = Describe("Context", func() {
 			Entry("no binding path specified", nil, defaultContainerPath),
 			Entry("binding path specified", &v1alpha1.BindingPath{ContainersPath: "foo.bar"}, "foo.bar"),
 		)
+		DescribeTable("should return slice of size 1 if application is specifiedt through label seclector", func(bindingPath *v1alpha1.BindingPath, expectedContainerPath string) {
```

Comment:
typo "specifiedt"

Reply:
it would be also more interesting to cover the case when there at least two applications with the same label - with that we check if the loop in https://github.com/redhat-developer/service-binding-operator/pull/971/files#diff-0168dc7ac8aa540873af78cf38f886f6a5ace89fb75dce9880490cedbed1d46eR157 really works.

Reply:
Done. 5326c6f674cb869fdaa8242bd4cc454fb72941cc